README.md

```diff
@@ -46,12 +46,18 @@ AND FURTHER DOWN IN `config/environment.js`:
 
 ```js
 if (environment === 'production') {
-  ENV.baseURL = '/';
+  ENV.rootURL = '/<name-of-git-repo>';
   ENV.locationType = 'hash';
-  ENV.apiHost = '<% replace with the URL to your deployed API %>'
+  ENV.apiHost = '<% replace with the URL to your deployed API %>';
 }
 ```
 
+1.  Now change the value of ENV.rootURL to be the name of your git repository; e.g. in the case of this repository it would be `ENV.rootURL = '/ember-deployment-guide'`
```

Comment:
@BenGitsCode I do not understand what you mean by

>So mine is working as '/'

Do you mean without the trailing `/`?  Something else?  I'm confused because your words don't seem to match the code you reference.